packages/vue-loader/index.js

```diff
@@ -32,10 +19,28 @@ export default {
     }
   },
   render(createElement) {
-    h = mdx.bind({ createElement, components: this.components })
-    return MDXContent({ components: this.components })
+    h = mdx.bind({createElement, components: this.components})
+    return MDXContent({components: this.components})
   }
 }
-   `
+`
+
+module.exports = async function (content) {
```

Comment:
for debug purposes it could be good to name the exported function

```suggestion
module.exports = async function mdxVueLoader(content) {
```